HandlerLifecycleManager.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace Symfony\Bundle\MonologBundle;
+
+
+use Monolog\Handler\HandlerInterface;
+
+class HandlerLifecycleManager
+{
+    /**
+     * @var HandlerInterface[]
+     */
+    private $handlers;
+
+    public function __construct(\IteratorAggregate $handlers)
```

Comment:
```suggestion
    public function __construct(iterable $handlers)
```

The type `IteratorAggregate` is a bit more specific that it needs to be, isn‘t it?

Reply:
the bundle supports PHP 5.6+ currently

Reply:
but `Traversable` should be used instead of `IteratorAggregate`

Reply:
we should drop support for SF 3.X, and bump PHP to 7.1
IMHO, we must do that before this PR